app/Filament/Pages/ClientTurnManager.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+namespace App\Filament\Pages;
+
+use App\Models\Client;
+use App\Models\Employee;
+use App\Enums\VisitDayEnum;
+use Carbon\Carbon;
+use Filament\Pages\Page;
+use Filament\Tables\Table;
+use Filament\Tables\Actions\Action;
+use Filament\Tables\Columns\TextColumn;
+use Filament\Tables\Concerns\InteractsWithTable;
+use Filament\Tables\Contracts\HasTable;
+use Filament\Tables\Filters\SelectFilter;
+use Filament\Support\Colors\Color;
+
+class ClientTurnManager extends Page implements HasTable
+{
+    use InteractsWithTable;
+
+    protected static ?string $navigationIcon = 'heroicon-o-queue-list';
+    protected static ?string $navigationLabel = 'Turnos de Visita';
+    protected static ?string $title = 'Gestión de Turnos de Visita';
+    protected static ?string $navigationGroup = 'Clientes';
+    protected static string $view = 'filament.pages.client-turn-manager';
+
+    protected function getDefaultVisitDay(): string
+    {
+        $dayMap = [
+            1 => VisitDayEnum::LUNES->value,
+            2 => VisitDayEnum::MARTES->value,
+            3 => VisitDayEnum::MIERCOLES->value,
+            4 => VisitDayEnum::JUEVES->value,
+            5 => VisitDayEnum::VIERNES->value,
+            6 => VisitDayEnum::SABADO->value,
+            7 => VisitDayEnum::LUNES->value, // Domingo muestra Lunes
+        ];
+
+        // Usar la zona horaria de Honduras
+        $today = Carbon::now('America/Tegucigalpa');
+        return $dayMap[$today->dayOfWeek];
+    }
+
+    public function table(Table $table): Table
+    {
+        $currentDay = $this->getDefaultVisitDay();
```

Comment:
Solo me queda la duda, saber donde usaste esto
```suggestion
        $currentDay = $this->getDefaultVisitDay();
```